### DISCUSSION TITLE
Toggle key please

### DISCUSSION THREAD

**Discussion by VallesDaGoat** [Fedoraware/Fedoraware#1190] [Ideas]
Currently using another training software JUST because it can make toggle-able keys. Ofc, fedoraware is better than that one, but the fact that theres no toggle-able keys in fedoraware makes me use the other thing. So please, add toggleable keys :D

**Comment by Ragdoll117** [comment]
Sorry brother, the pasters don't know how to make toggle keys sadly so you need to buy a rijin or nullcore for a key toggle.

**Comment by ghost** [comment]
i tried to make toggles but i accidentally toggled off my brian
